goggles.el

```diff
@@ -158,15 +158,18 @@ and activate/deactivate them separately."
        ,@(mapcar
           (lambda (f)
             `(defun ,(intern (format "goggles--adv-%s" f)) (orig &rest args)
-               (setq goggles--active (1+ goggles--active))
-               (unwind-protect (apply orig args)
-                 (setq goggles--active (- goggles--active 1)))))
+               (when (or goggles-mode goggles-global-mode)
```

Comment:
Why is the when conditional needed? I wonder if this is correct. If no googles mode is active, the original function is not called.